packages/protocol/contracts/governance/Election.sol

```diff
@@ -857,4 +905,56 @@ contract Election is
     }
     return res;
   }
+
+  // Struct to hold local variables for `forceDecrementVotes`.
+  // Needed to prevent solc error of "stack too deep" from too many local vars.
+  struct DecrementVotesInfo {
+    address[] groups;
+    uint256 remainingValue;
+  }
+
+  /**
+   * @notice Reduces the total amount of `account`'s voting gold by `value` by
+   *         iterating over all groups voted for by account.
+   * @param account Address to revoke votes from.
+   * @param value Amount of votes to revoke.
```

Comment:
It seems that `value` is not the votes to revoke, but the `max` votes to revoke.